refinery/dependency_migrations/chunked_upload/0001_initial.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
How was this migration created? 
Did `./manage.py makemigrations chunked_upload` not work?

Reply:
Btw, we exclude migrations from flake8: https://github.com/refinery-platform/refinery-platform/pull/2568/files